frontend/src/components/Facets/FacetsForm.tsx

```diff
@@ -107,6 +112,29 @@ const FacetsForm: React.FC<Props> = ({
                                 showArrow
                               >
                                 {facetOptions.map((variable) => {
+                                  let optionOutput:
+                                    | string
+                                    | React.ReactElement = (
+                                    <>
+                                      {variable[0]}
+                                      <span style={styles.facetCount}>
+                                        ({variable[1]})
+                                      </span>
+                                    </>
+                                  );
+                                  // The data node facet has a unique tooltip overlay to show the status of the highlighted node
+                                  if (facet === 'data_node' && nodeStatus) {
```

Comment:
Remove `&& nodeStatus`
The `StatusToolTip` component will handle if `nodeStatus` is available or not. If it is not available, it will display a question mark icon and the respective tooltip description.